terraform_compliance/extensions/terraform.py

```diff
@@ -416,6 +416,8 @@ def _mount_references(self):
                         target_resource = [k for k in self.resources.keys() if k.startswith(resource)]
 
                     for t_r in target_resource:
+                        if self.resources[t_r].get('values') is None:
+                            continue
```

Comment:
Wondering how you can hit this really. Can you also add a few functional test that will fail on this specific scenario please ?

Reply:
I got this from one of our deployments, I've been working to clean it up to make a test case from it

Reply:
@eerkunt ok cleaned it up and added in functional test that fails if you remove my change

Reply:
reverted the fix now that CI passed just to double check statement above is true in CI (worked for my local)

Reply:
hmmm now the CI won't kick off to prove it either way